kubens

```diff
@@ -24,6 +24,10 @@ IFS=$'\n\t'
 SELF_CMD="$0"
 KUBENS_DIR="${XDG_CACHE_HOME:-$HOME/.kube}/kubens"
 
+exec {STDIN}>&0
```

Comment:
can you please add a comment about what do these do? I'm not sure...

Reply:
Just pushed those comments you requested. These three lines assign the main process' standard streams to variables so that they can be used to invoke `kubectl auth can-i get namespaces` in a function, but with the main process' standard streams as its STDOUT, STDERR and STDIN. Using the main process' standard streams makes the `kubectl auth` call interactive, so client-go auth plugins are triggered